README.rst

```diff
@@ -103,114 +103,69 @@ Alternatively, you can explicitly start and end a span:
 Customization
 -------------
 
-Samplers
-~~~~~~~~
+There are several things you can customize how OpenCensus works:
```

Comment:
I'd say instead "several ways" or "several things you can customize to control how"

Reply:
Nice catch. Fixed.

Reply:
Changed to "There are several things you can customize in OpenCensus" since we're trying to list the customizable things right below.